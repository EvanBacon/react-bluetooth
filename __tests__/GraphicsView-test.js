import React from 'react';
import renderer from 'react-test-renderer';

import GraphicsViewIOS from '../lib/GraphicsView.ios.js';
import GraphicsViewDefault from '../lib/GraphicsView.js';

describe('GraphicsView', () => {
  describe('Platform.default', () => {
    it('renders', () => {
      const tree = renderer.create(<GraphicsViewDefault />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('Platform.ios', () => {
    it('renders', () => {
      const tree = renderer.create(<GraphicsViewIOS />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
