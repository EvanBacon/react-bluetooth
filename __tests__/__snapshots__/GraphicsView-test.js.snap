// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphicsView Platform.default renders 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    nativeRef_EXPERIMENTAL={[Function]}
    onContextCreate={[Function]}
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</View>
`;

exports[`GraphicsView Platform.ios renders 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    nativeRef_EXPERIMENTAL={[Function]}
    onContextCreate={[Function]}
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</View>
`;
